bool request
---
LogControl response
string data_location
string filename
